rules/integrations/google_workspace/collection_google_drive_ownership_transferred_via_google_workspace.toml

```diff
@@ -26,7 +26,48 @@ interval = "10m"
 language = "kuery"
 license = "Elastic License v2"
 name = "Google Drive Ownership Transferred via Google Workspace"
-note = """## Setup
+note = """## Triage and analysis
+
+### Investigating Google Drive Ownership Transferred via Google Workspace
+
+Google Drive is a cloud storage service available to users with a Google Workspace account that allows users to store and access files.
+
+Permissions to files or specific shared drives are typically established by a Google Workspace administrator based on roles and organizational units. Access to sensitive files and folders can be requested internally or externally where permission is granted by the owner. Adversaries abuse this trust system by accessing Google Drive resources with improperly scoped permissions and shared settings. Distribution of phishing emails is a common approach to also sharing malicious Google Drive documents where the recipient's Google Workspace privileges are inherited when ownership is granted by an external entity.
+
+This rule identifies when ownership of a shared drive, within a Google Workspace organization, is transferred to another internal user.
+
+#### Possible investigation steps
+
+- From the admin console, review admin logs for user accounts involved by searching in Security > Reporting > Audit and investigation > Admin log events
+- From user accounts involved, determine if accounts are active under Directory > Users
+- Check if the user accounts involved had been recently disabled and then re-enabled
+- Review user accounts involved for potentially misconfigured permissions or roles
+- Review the shared drive or files involved and related policies to determine if this action was expected and appropriate
+- If a shared drive, access requirments based on Organizational Units in Apps > Google Workspace > Drive and Docs > Manage shared drives
+- Triage potentially-related alerts from Security > Alert center based on the users involved
+
+### False positive analysis
+
+- Transferring drives requires Google Workspace administration permissions related to Google Drive. Review with the requester if this action was planned/expected and is appropriately targeting the correct receiver.
```

Comment:
```suggestion
- Transferring drives requires Google Workspace administration permissions related to Google Drive. Check if this action was planned/expected from the requester and is appropriately targeting the correct receiver.
```

Reply:
Feel free to reject this change if it's incorrect. I'm mainly suggesting it because it seems counterproductive to check with a user that's potentially an adversary.